tests/e2e/specs/settings/language.test.ts

```diff
@@ -43,11 +43,14 @@ describe('Settings - Language Settings Flow', () => {
       await drawerIcon.click();
     }
 
-    //const settingsInSpanish = await $(byTextMatches('Ajustes de la'));
-    // commenting out until translation done
-    const settingsInEnglish = await $(byTextMatches('Settings'));
-    await settingsInEnglish.click();
+    const english = $(byTextMatches('Settings'));
+    const spanish = $(byTextMatches('Ajustes de la'));
 
+    if (await spanish.isDisplayed()) {
+      await spanish.click();
+    } else {
+      await english.click();
+    }
```

Comment:
is this necessary if we know that there are spanish translations?

Reply:
But now that we have merged the translation into develop, if its not on your local branch, it means that translations are not properly synced to your local branch and so it should be failing. The other possibility is that it was taken out of the spanish translations, which should also be a failure because that shouldn't be happening